packages/breadboard-ui/src/types/types.ts

```diff
@@ -94,3 +94,26 @@ export type OutputArgs = {
     schema?: Schema;
   } & Record<string, unknown>;
 };
+
+export const BreadboardElementErrorCode = {
```

Comment:
If we subclass Event I don't think these would be necessary anymore, right?

Reply:
Yes. `BreadboardElementErrorCode` has been removed in the latest commit.